src/main/java/slave/Storage.java

```diff
@@ -0,0 +1,164 @@
+package slave;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * An object containing all the method
+ * Contains a reference to the taskList as well as the address of the save file
+ */
+public class Storage {
+    private String filePath;
+    private List<Task> list;
+
+    /**
+     * @param list     is the list of tasks which the user is editing
+     * @param filePath is the address of the text file to which any changes are saved to
+     */
+    public Storage(List<Task> list, String filePath) {
+        this.list = list;
+        this.filePath = filePath;
+    }
+
+    /**
+     * converts the taskList to a string format and writes it to the savefile
+     * every line contains only 1 task
+     * string format is as per the return value of toString() method of the respective task
+     * creates a new file at the specified address called "savefile.txt" in the event of a missing save file
```

Comment:
Can make use of some capitalisation here.

I.. shall not mention capitalisation anymore